docs/cookbook/emails/custom-email-template-per-channel.rst

```diff
@@ -0,0 +1,74 @@
+How to customise email templates per channel?
```

Comment:
Be consistent, sometimes you use "customi**z**e" and sometimes "customi**s**e" 😃

Reply:
When you're used to British English and realise the docs use American English 🤷‍♂